### PR TITLE
ISPN-12077 Serializer unable to serialize scattered caches

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.redhat.com/browse/ISPN-12077

### CODE REVIEW

File: core/src/main/java/org/infinispan/configuration/serializing/Serializer.java [hunk 1]
```diff
@@ -383,6 +386,15 @@ private void writeInvalidationCache(XMLExtendedStreamWriter writer, String name,
       writer.writeEndElement();
    }
 
+   private void writeScatteredCache(XMLExtendedStreamWriter writer, String name, Configuration configuration) throws XMLStreamException {
+      writer.writeStartElement(Element.SCATTERED_CACHE);
+      writer.writeAttribute(INVALIDATION_BATCH_SIZE, Integer.toString(configuration.clustering().invalidationBatchSize()));
```
Comment (pruivo) [inline core/src/main/java/org/infinispan/configuration/serializing/Serializer.java:391]:
```suggestion
      configuration.clustering().attributes().write(writer, ClusteringConfiguration.INVALIDATION_BATCH_SIZE, INVALIDATION_BATCH_SIZE);
```
it only serializes if the attribute changes.

How about the other attributes? `BIAS_ACQUISITION` and `BIAS_LIFESPAN` are configurable programmatically but they aren't parser...

Comment (ryanemerson) [reply core/src/main/java/org/infinispan/configuration/serializing/Serializer.java:391]: I just copied what was in the Parser. I can add these as well

Comment (pruivo) [reply core/src/main/java/org/infinispan/configuration/serializing/Serializer.java:391]: if you don't mind :) I hope it doesn't break anything xD

Comment (ryanemerson) [reply core/src/main/java/org/infinispan/configuration/serializing/Serializer.java:393]: I've added the missing attributes to the parser, serializer and the .xsd

**Comment by pruivo** [comment]
integrated! thanks @ryanemerson !

**Pull request merged**